lab1_task1b.html

```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE HTML>
```

Comment:
Better use `DOCTYPE` uppercase and `html` lowercase:
```<!DOCTYPE html>```

It is more generally accepted:
https://stackoverflow.com/a/9109157

Reply:
Do not overuse blank lines, only use them where it logically makes sense.
```html
        <tr>
           <td></td>
           <td rowspan="2" colspan="3" style="background-color: #ff5b2d "></td>
           <td colspan="2" style="background-color: #fffa69"></td>
           
          
        </tr>
```